cfme/tests/infrastructure/test_config_management.py

```diff
@@ -149,6 +149,29 @@ def test_config_system_tag(request, config_system, tag):
         "Failed to setup a configuration system's tag"
 
 
+@pytest.mark.tier(3)
+@pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type != "Ansible Tower")
+def test_ansible_tower_job_templates_tag(request, config_manager, tag):
+    """
+    Polarion:
+        assignee: anikifor
+        initialEstimate: 1/4h
+        casecomponent: Ansible
+        caseimportance: high
+
+    Bugzilla:
+        1673104
+    """
+    collection = config_manager.appliance.collections.ansible_tower_job_templates
+    job_template = collection.all()[0]
+    job_template.add_tag(tag=tag, details=False)
+    request.addfinalizer(lambda: job_template.remove_tag(tag=tag))
+    tags = job_template.get_tags()
+    assert '{}: {}'.format(tag.category.display_name, tag.display_name) in \
```

Comment:
Optional: Consider creating a new variable for `['{}: {}'.format(t.category.display_name, t.display_name) for t in tags]` to increase readability. Also why can't you do: 
```python,
assert tag in tags, "Failed to setup a job template's tag"
```

Reply:
@john-dupuy  Doesn't work that way
![image](https://user-images.githubusercontent.com/42433123/56744833-87e2c400-6779-11e9-9123-4af178918feb.png)